R/fit.R

```diff
@@ -11,11 +11,16 @@ generics::fit
 #' 
 #' @description
 #' Given the output of an infer core function, this function will fit
-#' a model using [stats::glm()] according to the formula and data supplied 
+#' a linear model using [stats::glm()] according to the formula and data supplied 
 #' earlier in the pipeline. If passed the output of [specify()] or 
 #' [hypothesize()], the function will fit one model. If passed the output 
 #' of [generate()], it will fit a model to each data resample, denoted in 
-#' the `replicate` column.
+#' the `replicate` column. The family of the fitted model depends on the type
+#' of the response variable. If the response is numeric, `fit()` will default
```

Comment:
Maybe something along the lines '... will use `family = "gaussian"` (ordinary linear regression)' and '... will use `family = "binomial"` (logistic regression)'?

Reply:
You got it! :-)